heartpoints-k8s.yml

```diff
@@ -34,10 +34,31 @@ spec:
           initialDelaySeconds: 10
           periodSeconds: 10
 ---
+apiVersion: networking.gke.io/v1beta1
```

Comment:
@bob-bins looks like we can declare two managed certs for GKE and give them unique names. it says [here](https://cloud.google.com/kubernetes-engine/docs/how-to/managed-certs) that each managed cert can only have one domain even though the property is called "domains" plural. So I am creating one for naked and one for www domain and naming appropriately.

Reply:
@aashreya hey i see some services hanging around from your earlier work, pics below:

![Screen Shot 2019-09-21 at 8 42 41 PM](https://user-images.githubusercontent.com/1911377/65381976-13bd3580-dcb1-11e9-89e5-acffd085186d.png)
![Screen Shot 2019-09-21 at 8 43 21 PM](https://user-images.githubusercontent.com/1911377/65381977-13bd3580-dcb1-11e9-9f81-1d963fe93d91.png)

Reply:
@aashreya maybe we can clean those up together or what do you think? i guess they got there thru manual process